src/diem/offchain/command.py

```diff
@@ -25,10 +41,32 @@ def reference_id(self) -> str:
         ...
 
     @abstractmethod
-    def new_command(self) -> "Command":
-        """sub-class should define **kwargs for creating new command from current instance"""
+    def new_request(self) -> CommandRequestObject:
         ...
 
     @abstractmethod
     def validate(self, prior: typing.Optional["Command"]) -> None:
         ...
+
+    @abstractmethod
+    def my_address(self) -> str:
+        """Returns my account identifier used for sending command request"""
+        ...
+
+    @abstractmethod
+    def opponent_address(self) -> str:
```

Comment:
maybe remote vs local?

Reply:
These 2 words were used in dip-1 v1 API url path: v1/<local>/<remote>/command
Hence I avoided them. 

I started to use "counter-party server/VASP" as the vasp/server a wallet app is communicating to. maybe switch to use counterparty_address instead.

Reply:
address this later